src/api/api.cc

```diff
@@ -11644,7 +11647,8 @@ extern "C" DLLEXPORT bool V8RecordReplayAllowSideEffects() {
 }
 
 extern "C" DLLEXPORT int V8RecordReplayNewDependencyGraphNode(const char* json) {
-  if (recordreplay::IsRecordingOrReplaying() && IsMainThread()) {
+  if (recordreplay::IsRecordingOrReplaying() && IsMainThread() &&
```

Comment:
Repeated condition longer than one line: Please re-write as function 🙏
(You gave me a similar comment a while ago)

Reply:
You can re-use it in 5 places, if I'm not miscounting.